applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -596,9 +614,35 @@ int unicast_server_send(struct le_audio_encoded_audio enc_audio)
 {
 #if (CONFIG_BT_AUDIO_TX)
 	int ret;
+	uint8_t num_active_streams = 0;
 
-	ret = bt_le_audio_tx_send(bap_tx_streams, audio_mapping_mask, enc_audio,
-				  CONFIG_BT_ASCS_ASE_SRC_COUNT);
+	struct le_audio_tx_info tx[CONFIG_BT_ASCS_ASE_SRC_COUNT];
+
+	for (int i = 0; i < ARRAY_SIZE(cap_tx_streams); i++) {
+		if (!le_audio_ep_state_check(cap_tx_streams[i]->bap_stream.ep,
+					     BT_BAP_EP_STATE_STREAMING)) {
+			continue;
+		}
+
+		/* Set cap stream pointer */
+		tx[num_active_streams].cap_stream = cap_tx_streams[i];
+
+		/* Set index */
+		tx[num_active_streams].idx.lvl1_idx = 0;
+		tx[num_active_streams].idx.lvl2_idx = 0;
+		tx[num_active_streams].idx.lvl3_idx = i;
+
+		/* Set channel location */
+		/* TODO: Use the function below once
+		 * https://github.com/zephyrproject-rtos/zephyr/pull/72908 is merged
+		 */
+		/* tx[num_active_streams].audio_channel = audio_map_location_get(stream);*/
+		tx[num_active_streams].audio_channel = AUDIO_CH_L;
```

Comment:
Comment why this is always set to AUDIO_CH_L

Reply:
We don't really have a good reason, do we? Except that the hardware is limited to mono microphone. Should we instead have a define called `AUDIO_CH_MIC = 0`?